### DISCUSSION TITLE
How do we can publish our defi SOL wallet on Wallet Connect?

### DISCUSSION THREAD

**Discussion by innersystem** [WalletConnect/walletconnect-monorepo#1016] [Developer Support]
Hi! Please hope some1 can help me)
We developed solana mobile wallet and implemented Wallet Connect beta (with SOL) there.
Now we want to publish our wallet through Wallet Connect Cloud, and I cant choose any SOL chains. 

How can I make it?

**Comment by chadyj** [comment]
Sol chains have been added.
Thanks for your feedback.

**Reply by nooxx** [reply]
@chadyj Is this documented anywhere?

**Reply by chadyj** [reply]
Take a look at https://docs.walletconnect.com/2.0/
